e2e/wdio/core/pages/object-identifier.po.ts

```diff
@@ -0,0 +1,28 @@
+import { CoreBaseComponentPo } from './core-base-component.po';
+import { waitForElDisplayed, waitForPresent } from '../../driver/wdio';
+
+export class ObjectIdentifierPo extends CoreBaseComponentPo {
+    private url = '/object-identifier';
+
+
+    identifier = '.fd-object-identifier';
+    clickableLinks = 'fd-link fd-object-identifier__link';
```

Comment:
make selector shorter, can be just '.fd-link'

Reply:
Done